packages/actor-init-query/lib/VoidMetadataEmitter.ts

```diff
@@ -86,6 +86,32 @@ export class VoidMetadataEmitter {
     return quads;
   }
 
+  private convertValue(
+    key: string,
+    value: string,
+  ): string {
+    const xsd = 'http://www.w3.org/2001/XMLSchema#';
+    const stringLiterals = new Set([ 'alternative', 'description', 'title' ]);
+    const dateLiterals = new Set([
```

Comment:
Let's make these static final in the class, to avoid re-creating them every call.